holidays/countries/gambia.py

```
@@ -0,0 +1,90 @@
+            gregorian_eid_fitr = Gregorian(hijri_year_start + (year - 2000), 10, 1).to_gregorian()
+            manual_holidays[gregorian_eid_fitr] = "Eid al-Fitr (Approximate)"
+
+            # Eid al-Adha (Feast of Sacrifice) - roughly 12th month, 10th day
+            gregorian_eid_adha = Gregorian(hijri_year_start + (year - 2000), 12, 10).to_gregorian()
+            manual_holidays[gregorian_eid_adha] = "Eid al-Adha (Approximate)"
+
+            # Mawlid (Prophet's Birthday) - roughly 3rd month, 12th day
+            gregorian_mawlid = Gregorian(hijri_year_start + (year - 2000), 3, 12).to_gregorian()
+            manual_holidays[gregorian_mawlid] = "The Prophet's Birthday (Approximate)"
+    except Exception as e:
+        print(f"Error calculating Islamic holidays for {year}: {e}")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)